### PR TITLE
MODDCB-19 Refactoring yaml files

### PR DESCRIPTION
**Pull request by Vignesh-kalyanasundaram**
## Purpose
The purpose of the PR is to refactor yaml files
_Describe the purpose of the pull request. Include background information if necessary._

## Approach
_How does this change fulfill the purpose?_

#### TODOS and Open Questions
- [ ] Use GitHub checklists. When solved, check the box and explain the answer.

## Learning
_Describe the research stage. Add links to blog posts, patterns, libraries or addons used to solve this problem._

### CODE REVIEW

File: src/main/resources/swagger.api/schemas/transactionStatus.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+TransactionStatus:
+  type: object
+  properties:
+    status:
+      type: string
+      enum:
+        - CREATED #Created by DCB
+        - OPEN #Created in lending library, Item is available
+        - CANCELLED #Request canceled by lending library or patron
+        - IN_TRANSIT #checked in at another service point
```
Comment (gurleenkaurbp) [inline src/main/resources/swagger.api/schemas/transactionStatus.yaml:10]: @kalibek-epam 
Does this state IN_TRANSIT and IN_TRANSIT_TO_LENDING still exists and be present in the yaml file?

File: src/main/resources/swagger.api/dcb_transaction.yaml [hunk 2]
```diff
@@ -6,148 +6,124 @@ info:
...
-        required: true
-        schema:
-          type: string
     post:
       description: Create circulation request
       operationId: createCirculationRequest
       tags:
         - circulation
+      parameters:
+        - $ref: '#/components/parameters/dcbTransactionId'
```
Comment (gurleenkaurbp) [inline src/main/resources/swagger.api/dcb_transaction.yaml:15]: Hi @Vignesh-kalyanasundaram ,
As per the ongoing discussion there is going to be a library role as part of the POST request. Please keep that provision over here in the POST request. 
CC: @steveellis , @kalibek-epam

File: src/main/resources/swagger.api/dcb_transaction.yaml [hunk 3]
```diff
@@ -6,148 +6,124 @@ info:
...
-              schema:
-                $ref: '#/components/schemas/TransactionStatusResponse'
-
+          $ref: '#/components/responses/TransactionStatusResponse'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
```
Comment (gurleenkaurbp) [inline src/main/resources/swagger.api/dcb_transaction.yaml:59]: I am just thinking upon if there are any more error codes that we should add. @kalibek-epam  has K-int provided a list of error codes that we have to match up with or these many are enough at the moment ?

**Review by gurleenkaurbp — APPROVED** [review]
I have some comments in the PR but still approving it.

**Review by SinghAdes — APPROVED** [review]

**Review by DmytroBykov1 — APPROVED** [review]

**Review by MagzhanArtykov — APPROVED** [review]

**Pull request merged**